runtime-modules/proposals/codex/src/tests/mock.rs

```diff
@@ -105,6 +104,7 @@ frame_support::construct_runtime!(
         OperationsWorkingGroupBeta: working_group::<Instance7>::{Pallet, Call, Storage, Event<T>},
         OperationsWorkingGroupGamma: working_group::<Instance8>::{Pallet, Call, Storage, Event<T>},
         DistributionWorkingGroup: working_group::<Instance9>::{Pallet, Call, Storage, Event<T>},
+        Council: council::{Pallet, Call, Storage, Event<T>},
```

Comment:
Why did you find it necessary to move the Council pallet down here.

Reply:
```
error[E0277]: the trait bound `RuntimeEvent: From<pallet_council::RawEvent<u64, u64, u64, u64>>` is not satisfied
   --> runtime-modules/proposals/codex/src/tests/mock.rs:696:25
    |
696 |     type RuntimeEvent = RuntimeEvent;
    |                         ^^^^^^^^^^^^ the trait `From<pallet_council::RawEvent<u64, u64, u64, u64>>` is not implemented for `RuntimeEvent`
```

Reply:
without that